1.7/administration/high-availability/multi-region.md

```diff
@@ -0,0 +1,33 @@
+---
+post_title: Multiple Regions
+menu_order: 2
+---
+
+DC/OS supports multiple region configurations. This topic describes the setup recommendations and caveats.
+
+# Single Region Masters and Cross-Region Agents 
```

Comment:
While this is possible, it definitely isn't tested and not recommended. Maybe we should highlight that more?